### PR TITLE
Longest running tests report

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: apps/report/views.py [hunk 1]
```diff
@@ -58,6 +61,7 @@ def get_context_data(self, **kwargs):
             "group__name").annotate(dcount=Count("group")).order_by("-dcount")
         context["repotest"] = Test.objects.values("git__name").annotate(
             dcount=Count("git")).order_by("-dcount")
+        context["testlengths"] = Task.objects.filter(datestart__gte = datetime.now()+timedelta(hours=-168)).values("test__name").annotate(avg_duration=Avg('duration')).order_by("-avg_duration")
```
Comment (Pajinek) [inline apps/report/views.py]: pep8 missing - https://www.python.org/dev/peps/pep-0008/#id19

Comment (lhellebr) [inline apps/report/views.py]: Fixed pep8 warnings in my newly added code. There are 2 more warnings in code unrelated to this PR.

File: apps/report/views.py [hunk 2]
```diff
@@ -4,7 +4,10 @@
 # Author: Pavel Studenik <pstudeni@redhat.com>
 # Date: 3.2.2016
 
-from django.db.models import Count
+from datetime import datetime, timedelta
+
+from django.utils import timezone
```
Comment (Pajinek) [inline apps/report/views.py]: why did you import timezone?

File: apps/report/views.py [hunk 3]
```diff
@@ -58,6 +61,10 @@ def get_context_data(self, **kwargs):
             "group__name").annotate(dcount=Count("group")).order_by("-dcount")
         context["repotest"] = Test.objects.values("git__name").annotate(
             dcount=Count("git")).order_by("-dcount")
+        context["testlengths"] = Task.objects.filter(
+            datestart__gte=datetime.now()+timedelta(hours=-168)).values(
```
Comment (Pajinek) [inline apps/report/views.py]: And here you used datetime.

File: apps/report/views.py (hunk 2 above)
Comment (lhellebr) [inline apps/report/views.py]: Probably some leftover from testing; removed.

**Pull request merged**